package.json

```diff
@@ -153,7 +153,7 @@
     "color-name": "1.1.3"
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
+    "vscode:prepublish": "npm run compile && npx typedoc --out docs --entryPointStrategy expand src/",
```

Comment:
Since these docs are for people *developing* this extension, and thus the ones *not* using the published version, I don't think it makes sense to run it as part of the pre-publish step. I think it'd be sufficient to have an `npm run docs` script that generates them.

If you think otherwise please let me know why!